ht.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+
+
+    <!-- Image Avatar -->
+
+    <figure class="avatar avatar-md">
+        <img class="avatar-img" src="...." alt="....">
+    </figure>
+
+    <!-- Letter Avatar -->
+
+    <figure class="avatar avatar-letter avatar-sm warning ">
+        <span>RK</span>
+    </figure>
+
+    <!-- Icon Avatar -->
+
+    <!-- SVG Image  -->
+    <figure class="avatar avatar-icon  avatar-sm primary">
+        <img class="avatar-icon" src="./camera.svg" alt="">
```

Comment:
give a short info in  `alt` everywhere

Reply:
I forgot to remove that file, that is for testing. thanks

Reply:
fixed - d024da009973ecafa0fe044faac36624c2778267